WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -350,6 +350,10 @@ decisions = {
 		effect = {
 			set_global_flag = rome_restored
 			narrative_event = { id = ancrel.2220 tooltip = EVTTOOLTIPLoR.20 }
+			add_law = succ_primogeniture
+			add_law = vice_royalty_2
+			add_law = imperial_administration
+			set_allow_free_vice_royalty_revokation = yes
```

Comment:
@loup99  

Okay. That still doesn't address why the two Roman Empires don't have the Imperial Administration Law in effect at the game's start, like they would have had in RL. What I wrote about _uniting_ (not restoring) the two Roman Empires was a makeshift quick fix because I had no idea how to code the two empires to start with Imperial Administration.

I'll revert it if you'd like.